tests/testlogin.js

```diff
@@ -0,0 +1,91 @@
+//const { assert } = require("console");
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+
+describe ('TestLogin', () => {
+
+   
+    it('Login  with correct username and correct password', async () => {
+
+        var expect = require('chai').expect;
```

Comment:
**expect()** is globally defined in newer versions of webdriverio, there's no need to explicitly require it. Delete this line